src/main/java/org/jabref/gui/openoffice/OpenOfficePanel.java

```diff
@@ -657,4 +661,15 @@ private ContextMenu createSettingsPopup() {
 
         return contextMenu;
     }
+
+    private void addAlwaysCitedOnPagesTextOption(ContextMenu contextMenu) {
+        OpenOfficePreferences openOfficePreferences = preferencesService.getOpenOfficePreferences();
+
+        CheckMenuItem alwaysAddCitedOnPagesText = new CheckMenuItem(Localization.lang("Automatically add \"Cited on pages...\" at beginning of citation"));
```

Comment:
Good job. It definitely should be at the end of the citation. "[backref](https://ctan.org/pkg/backref?lang=en)", which is part of the hyperref package offers the "cited on pages" feature for LaTeX documents, so that could serve as an inspiration. I do not remember exactly, but I believe I made some minor adjustments to the default backref settings to always start in a new line, which I think looks slick :-)

Here an example from a paper I wrote in LaTeX that includes the backref feature:
![image](https://github.com/user-attachments/assets/de90f183-477b-4e07-8568-959316547e0e)